src/antq/util/maven.clj

```diff
@@ -79,14 +79,23 @@
      :artifact artifact
      :remote-repos remote-repos}))
 
-(defn ^Model read-pom
+(defn- ^Model read-pom*
   [^String url]
   (with-open [reader (io/reader url)]
     (.read (MavenXpp3Reader.) reader)))
 
-(defn ^String get-url
-  [^Model model]
-  (.getUrl model))
```

Comment:
@danieroux `get-url` is required yet. Is this deletion mistake?

Reply:
rebase gone wrong!